plugins/gocd/src/components/GoCdBuildsTable/GoCdBuildsTable.tsx

```diff
@@ -244,9 +244,11 @@ export const GoCdBuildsTable = (props: GoCdBuildsProps): JSX.Element => {
             search: true,
             searchAutoFocus: true,
             debounceInterval: 800,
-            paging: false,
+            paging: true,
+            padding: 'dense',
```

Comment:
Since now we are rendering potentially way more builds, let's make the table `dense`.